script/c513000138.lua

```diff
@@ -1,58 +1,64 @@
---The Wicked Avatar (Anime)
---邪神アバター
---マイケル・ローレンス・ディーによってスクリプト
---scripted by MLD
---credit to TPD & Cybercatman
---updated by Larry126
-function c513000138.initial_effect(c)
-	aux.CallToken(421)
-	--summon with 3 tribute
-	local e1=Effect.CreateEffect(c)
-	e1:SetProperty(EFFECT_FLAG_CANNOT_DISABLE+EFFECT_FLAG_UNCOPYABLE)
-	e1:SetType(EFFECT_TYPE_SINGLE)
-	e1:SetCode(EFFECT_LIMIT_SUMMON_PROC)
-	e1:SetCondition(c513000138.ttcon)
-	e1:SetOperation(c513000138.ttop)
-	e1:SetValue(SUMMON_TYPE_ADVANCE)
-	c:RegisterEffect(e1)
-	local e2=e1:Clone()
-	e2:SetCode(EFFECT_LIMIT_SET_PROC)
-	c:RegisterEffect(e2)
-	--atk
-	local e3=Effect.CreateEffect(c)
-	e3:SetType(EFFECT_TYPE_SINGLE)
-	e3:SetCode(EFFECT_SET_ATTACK_FINAL)
-	e3:SetProperty(EFFECT_FLAG_SINGLE_RANGE+EFFECT_FLAG_REPEAT+EFFECT_FLAG_DELAY)
-	e3:SetRange(LOCATION_MZONE)
-	e3:SetValue(c513000138.adval)
-	c:RegisterEffect(e3)
-	local e4=e3:Clone()
-	e4:SetCode(EFFECT_SET_DEFENSE_FINAL)
-	c:RegisterEffect(e4)
-end
--------------------------------------------------------------------
-function c513000138.ttcon(e,c)
-	if c==nil then return true end
-	return Duel.GetLocationCount(c:GetControler(),LOCATION_MZONE)>-3 and Duel.GetTributeCount(c)>=3
-end
-function c513000138.ttop(e,tp,eg,ep,ev,re,r,rp,c)
-	local g=Duel.SelectTribute(tp,c,3,3)
-	c:SetMaterial(g)
-	Duel.Release(g,REASON_SUMMON+REASON_MATERIAL)
-end
-function c513000138.filter(c)
-	return c:IsFaceup() and c:GetCode()~=21208154
-end
-function c513000138.adval(e,c)
-	local g=Duel.GetMatchingGroup(c513000138.filter,0,LOCATION_MZONE,LOCATION_MZONE,nil)
-	if g:GetCount()==0 then 
-		return 1
-	else
-		local tg,val=g:GetMaxGroup(Card.GetAttack)
-		if val>=9999999 then
-			return val
-		else
-			return val+1
-		end
-	end
+--The Wicked Avatar (Anime)
+--邪神アバター
+--マイケル・ローレンス・ディーによってスクリプト
+--scripted by MLD
+--credit to TPD & Cybercatman
+--updated by Larry126
+Duel.LoadScript("c421.lua")
+local s,id=GetID()
+function s.initial_effect(c)
+	--summon with 3 tribute
+	local e1=Effect.CreateEffect(c)
+	e1:SetProperty(EFFECT_FLAG_CANNOT_DISABLE+EFFECT_FLAG_UNCOPYABLE)
+	e1:SetType(EFFECT_TYPE_SINGLE)
+	e1:SetCode(EFFECT_LIMIT_SUMMON_PROC)
+	e1:SetCondition(s.ttcon)
+	e1:SetOperation(s.ttop)
+	e1:SetValue(SUMMON_TYPE_TRIBUTE)
```

Comment:
This constant is not maped in Ygopro Percy. Should be reverted back to **SUMMON_TYPE_TRIBUTE**